releasenotes/notes/fix-stat-flags-de59f48fea4ce5c2.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Accommodate cases where stat flags of ``/proc/<pid>/stat`` exceed int32.
```

Comment:
Let's be more explicit in the description of the problem. Something like "Fix to the process-agent from picking up  processes which are kernel threads due integer overflow when parsking '/proc/<pid>/stat"